ruby_on_rails/app_initialisation.md

```diff
@@ -1,61 +1,77 @@
 # Initialise the Rails App
 
-* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v` and update it if you are not.
-Run `rbenv versions` to check your installed ruby versions.
+## Default Rails setup
+
+* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v`
+and update it if you are not. Run `rbenv versions` to check your installed ruby versions.
 Follow the instructions on screen to eventually install it.
 
+* Switch the global Ruby version to the newest one with `rbenv global <ruby-version>` so that the latter commands
+run in the correct context.
+
 * Run `gem install bundler` to install Bundler.
 You may have it already installed, but this command will make sure that you have the latest version.
 
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
-You can do this with `gem update rails`.
+You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] -d postgresql --skip-sprockets --webpack` where the `project-name` is exactly the one you chose before.
+* Start a new Rails project using `rails new [project-name] --database=postgresql --skip-test --skip-sprockets --webpacker` where the `project-name` is exactly the one you chose before.
 You may want to choose a different database from Postgres, but most of the time that will be your choice.
 If you do not need a DB you may rethink the fact that you may not need Rails at all :) Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)
 
-* Specify the ruby version in the project's `Gemfile` with
+* Update the Ruby version in `.ruby-version` and load it in the fresh project's `Gemfile` like this
 
-```ruby File.read(File.join(__dir__, '.ruby-version'))```
+  ```ruby File.read(File.join(__dir__, '.ruby-version'))```
 
-[This is used by Heroku to determine what version to use.](https://devcenter.heroku.com/articles/ruby-versions)
+  [This is used by Heroku to determine what version to use.](https://devcenter.heroku.com/articles/ruby-versions)
 
-## bin scripts
+* Run `bin/setup`
+* Then check your default Rails setup by running `rails s` and visiting `localhost:3000`.
+  You should be on Rails now, yay!
```

Comment:
I like this part. Maybe `[project-name].localhost:3000`? This would also make sure that you have something configured like hotel and is correctly working

Reply:
I'd not. People who have hotel, know it probably, right?